src/wallet/walletdb.cpp

```diff
@@ -1170,6 +1199,9 @@ DBErrors WalletBatch::LoadWallet(CWallet* pwallet)
         }
 #endif
 
+        // Load decryption keys
+        result = std::max(LoadDecryptionKeys(pwallet, *m_batch), result);
+
```

Comment:
In 8bb7a8b0:

This is moved without an explanation. Could mention the rationale in the commit description.

Reply:
Moved this to the commit where it actually matters, so should hopefully make more sense.